js/screens/OnBoarding/styles.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet, Dimensions } from 'react-native';
+const { width, height } = Dimensions.get('window');
+import { fonts } from '../../config/styles';
+
+const styles = StyleSheet.create({
+  images: {
+    height: height * 0.29,
+    width: width * 0.55,
+  },
+  text: {
+    color: '#0072A0',
+    marginTop: -90,
+    paddingLeft: 40,
+    paddingRight: 40,
+    textAlign: 'left',
+    fontFamily: fonts.main,
+  },
+  lastImage: {
+    height: height * 0.29,
+    width: width * 0.55,
```

Comment:
again, how did you come up with these magic numbers? Please let me know.

Reply:
hi, i inspected the invision width and height, got the percentage to the phone dimensions they had then applied it to mine